ts/Data/AJAX/Authenticators/AuthenticatorType.d.ts

```diff
@@ -0,0 +1,31 @@
+/* *
+ *
+ *  Data Layer
+ *
+ *  (c) 2012-2020 Torstein Honsi
```

Comment:
I didn't write this... And not in 2012 or 2020.

If determining the real author for the files is not straight forward, we should write `(c) 2020-2021 Highsoft`